src/init.cc

```diff
@@ -2200,6 +2200,17 @@ static ncclResult_t ncclCommInitRankDev(ncclComm_t* newcomm, int nranks, ncclUni
   NCCLCHECKGOTO(ncclAsyncLaunch(&job->base, ncclCommInitRankFunc, NULL, free, comm), res, fail);
 
 exit:
+  // for loggin only, not ready for replaying
+  // !recording at sink
+  CollectiveCall initCall;
+  initCall.coll = rrCommInitRank;
+  initCall.comm = comm;
+  initCall.nRanks = nranks;
+  initCall.globalRank = myrank;
+  initCall.hipDev = cudaDev; // ...?
+  initCall.name = "CommInitRank";
```

Comment:
Move this the helper CollectiveCall constructor function for now in Replayer.  Try not to overly modify original source code file / make sure that it's evident that this is a RCCL addition.